cmake/configure/configure_10_mpi.cmake

```diff
@@ -65,8 +65,12 @@ macro(feature_mpi_configure_external)
   # (in Modules/FindMPI.cmake) at some point. For the time being this is an
   # advanced configuration option.
   #
-  option(DEAL_II_MPI_WITH_CUDA_SUPPORT "Enable MPI Cuda support" OFF)
-  mark_as_advanced(DEAL_II_MPI_WITH_CUDA_SUPPORT)
+  if(DEAL_II_MPI_WITH_CUDA_SUPPORT)
```

Comment:
Where does `DEAL_II_MPI_WITH_CUDA_SUPPORT` get set? AFAICT, based on the changes to `cuda.html`, we would not expect this to be set by a user any more.

Reply:
Only for backward-compatibility.